util/util.go

```diff
@@ -40,6 +44,19 @@ func ExpandPath(path string) (string, error) {
 	return filepath.Join(usr.HomeDir, path[1:]), nil
 }
 
+func findAlias(path string) (string, error) {
```

Comment:
We should probably memoize this function? So that's loaded in memory at will and we don't have to go to the disk again

Reply:
I need to ask you on memoize next week

Reply:
Right now I added a `var map[string]string` in the utils package. Do you think this is a good solution?